src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
+  const stringModified = sourceString
+    .split(/;\s*/)
+    .filter((item) => item.trim() !== '');
+
+  stringModified.forEach((item) => {
+    const index = item.indexOf(':');
```

Comment:
```suggestion
    const [key, value] = item.split(':');
```
You can just split instead